src/components/black-friday/payment/PriceCard.tsx

```diff
@@ -105,7 +105,7 @@ export default function PriceCard({
             `}
           >
             <p className={` flex flex-row items-start space-x-1 whitespace-nowrap font-medium text-white`}>
-              <span className={`currency ${price <= 0 ? 'hidden' : ''}`}>{country}</span>
+              <span className={`currency ${price <= 0 ? 'hidden' : ''}`}>{currency}</span>
               <span className="text-4xl font-bold">
                 {price <= 0 ? `${contentText.freePlan}` : planType === 'business' ? convertedPrice : convertedPrice}
```

Comment:
A tip:

Make your components as dumb as you can, so you only use them to mostly display data.

Give your component ready to display data, and store the logic to define which data should you display in services or utils files, that way if you need to change your logic, you change it in your centralized file, instead of UI components.

An example:

` {price <= 0 ? `${contentText.freePlan}` : planType === 'business' ? convertedPrice : convertedPrice}`

can be instead:

`isFreePlan ? "FREE_PLAN" :  convertedPrice`
`isBusinessPlan ? "whatever" : "whatever2`

Also don't use nested ternary operators, if you need complex conditional logic, use a function with if/else or something like that, nested ternary operators are hard to read and even harder inside a component